### ISSUE TITLE
Tidy dispersion parameters? 

### ISSUE THREAD

**Issue by bwiernik** [bbolker/broom.mixed#107]
Would it be possible to add dispersion parameter information to the tidy output? Ideally, it would also be nice to have the option to exponentiated and exponentiate+square root transform the estimates and confidence intervals.

**Comment by bbolker** [comment]
In principle using `cond=c("cond","zi","disp")` should work (haven't tested recently)?  There is an `exponentiate` argument as well (also may not be well-tested).  Out of curiosity, what would be the use-case for an exp+sqrt transform?

**Comment by bwiernik** [comment]
Currently, there is a check that errors if `cond` contains something other than "cond" or "zi". 

For the `exponentiate` argument, it'd be nice to be able to specify it separately for cond vs zi vs disp. For example, if I fit a linear regression model with a dispersion model, I'd like to exponentiate the disp parameters, but not the cond parameters.

For the  exp+sqrt transform, for a linear regression model like I describe above, the dispersion parameter are in log(variance) units. An exp+sqrt transform expresses them in (multiplicative) standard deviation units.

**Comment by nthun** [comment]
@bbolker it seems like the `condition = "disp"` doesn't work currently:

`Error in tidy.glmmTMB(mean_model, component = c("cond", "disp")) : 
  only works for conditional and (partly for) zero-inflation components`

**Comment by mgirdwood7** [comment]
Yes, confirmed. In the source code for ```tidy.mixed```, ```"disp"``` is not an available option. I created a workaround in the mean time:

i edited the ```tidy.mixed``` function to add ```"disp"``` to this line of the function:

```
if (length(component[!component %in% c("cond", "zi")]) > 0L) {
```

The function then works. It doesn't return confidence intervals for a similar reason I'm guessing. Again i made a quick workaround by creating a helper function:

```
tidy_betab <- function(model, comp){
  broomframe <- tidy(model, component = comp) # the original data frame from broom.mixed
  confintframe <- confint(model, estimate = FALSE) %>%  # gets the confidence interval data frame
    as.data.frame() %>% setNames(c("conf.low", "conf.high"))
    
  out <- bind_cols(broomframe, confintframe)
  return(out)
}
```

Thank you for this very helpful package!!
